ipywidgets/static/widgets/js/manager.js

```diff
@@ -332,19 +348,19 @@ define([
          *      widget_class: (optional) string
          *          Target name of the widget in the back-end.
          *      comm: (optional) Comm
+         *           Comm object associated with the widget.
+         *      model_id: (optional) string
+         *           If not provided, the comm id is used.
          *
-         * Create a comm if it wasn't provided.
+         * Either a comm or a model_id must be provided.
```

Comment:
Or just renaming and keeping `create_model`.  I do think having both `create_model` and `new_model` would be confusing.